src/Core/Components/Icons/Icon.cs

```diff
@@ -92,7 +92,7 @@ public virtual MarkupString ToMarkup(string? size = null, string? color = null)
         {
             var styleWidth = size ?? $"{(int)Size}px";
             var styleColor = color ?? Color ?? "var(--accent-fill-rest)";
-            return new MarkupString($"<svg viewBox=\"0 0 {(int)Size} {(int)Size}\" fill=\"{styleColor}\" style=\"background-color: var(--neutral-layer-1); width: {styleWidth};\" aria-hidden=\"true\">{Content}</svg>");
+            return new MarkupString($"<svg viewBox=\"0 0 {(int)Size} {(int)Size}\" width=\"{styleWidth}\" fill=\"{styleColor}\" style=\"background-color: var(--neutral-layer-1); width: {styleWidth};\" aria-hidden=\"true\">{Content}</svg>");
```

Comment:
Why do you need to add the `width` attribute (we already have this attribute in `style`)?

Reply:
The intrinsic size of the svg is set to 150px (as per specification). Supplying the width attribute overrides this. The style attribute seems to not work (at least in Chrome)